mayastor-test/e2e/common/io_connect_node.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Script to disconnect a node from another node using iptables
+# $1 is the node-name to isolate/restore
+# $2 is the other node-name
+# $3 is "I" to disconnect, "D" to reconnect
+# assumes the nodes use a known fixed set of IP addresses and node names
+
+# edit the line below, if necessary
+KUBESPRAY_REPO="$HOME/work/kubespray"
```

Comment:
```suggestion
KUBESPRAY_REPO="${KUBESPRAY_REPO:-$HOME/work/kubespray}"
```

This would allow people to `export KUBESPRAY_REPO=...` without needing to edit script (and will allow them to git pull changes more easily).